doc/nrf/ug_dev_model.rst

```diff
@@ -357,15 +357,68 @@ This workflow is particularly beneficial if your application is split among mult
 
 In order to implement this approach you first need to create a manifest repository of your own, which just means a repository that contains a :file:`west.yml` manifest file in its root.
 Next you must populate the manifest file with the list of repositories and their revisions.
-In general, the easiest thing to do is to take the :file:`west.yml` in `fw-nrfconnect-nrf`_ and copy its entries directly as a starting point.
+
+In general, the easiest thing to do is to import the :file:`west.yml` in `fw-nrfconnect-nrf`_ using west's manifest imports feature, like this:
+
```

Comment:
Perhaps add a sentence here that clearly states that all the NCS projects, includint Zephyr's, will be automatically added to your manifest in this manner.